src/endpoints/getPlayerByName.ts

```diff
@@ -0,0 +1,33 @@
+import { HLTVConfig } from '../config'
+import axios from 'axios'
+import { ISOCodesToLong } from '../utils/parsing'
+import { SearchPlayer } from 'models/SearchPlayer'
+
+export const getPlayerByName = (config: HLTVConfig) => async ({
+  name
+}: {
+  name: string
+}): Promise<SearchPlayer> => {
+  return axios.get(`${config.hltvUrl}/search?term=${name}`)
+    .then((response) => {
+      let playerArr = response.data[0].players
+
+      return playerArr.map(element => {
+        const { flagUrl } = element
+        return {
+          id: element.id,
```

Comment:
Why not call `HLTV.getPlayer` with the ID from the response instead? That way you will get way more information and also `ISOCodesToLong` becomes unneeded.

Reply:
Yes but you would need too do another request. That is why i choose to do it that way.
But yes its better that way i think. I'm going too Change it.